go/pkg/filemetadata/filemetadata_test.go

```diff
@@ -118,6 +123,84 @@ func TestComputeFilesWithXattr(t *testing.T) {
 	}
 }
 
+func TestComputeFileDigestWithXattr(t *testing.T) {
+	xattrDgName := "user.myhash"
+	overwriteXattrDgName(t, xattrDgName)
+	tests := []struct {
+		filename   string
+		contents   string
+		xattrDgStr string
+		wantDgStr  string
+		wantErr    bool
+	}{
+		{
+			filename:  "provide_no_xattr_will_generate_real_digest(sha256sum+real_size)",
```

Comment:
you can add a `name` property with a more readable short name that's not necessarily the file name. This is the convention in the rest of the code for the test name.